addons/im_livechat/models/discuss_channel.py

```diff
@@ -58,8 +58,8 @@ def _to_store_defaults(self, for_current_user=True):
             "anonymous_name",
             "chatbot_current_step",
             "livechat_active",
-            Store.One("country_id", ["code", "name"], rename="anonymous_country"),
             "livechat_operator_id",
+            Store.One("country_id", ["code", "name"]),
```

Comment:
I would have it alphabetical by field name in this case, so below `chatbot_current_step`